src/test/java/system_tests/HttpHandler.java

```diff
@@ -0,0 +1,41 @@
+package system_tests;
+
+import okhttp3.*;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class HttpHandler {
+  public static Response postRequest(String endPoint, HashMap<String, String> bodyParam) {
+    OkHttpClient client = new OkHttpClient().newBuilder().build();
+    MediaType mediaType = MediaType.parse("application/json");
+    RequestBody body = RequestBody.create(mediaType, new JSONObject(bodyParam).toString());
+    Request request =
+        new Request.Builder()
+            .url(endPoint)
+            .method("POST", body)
+            .addHeader("Content-Type", "application/json")
+            .build();
+
+    Response response;
+    try {
+      response = client.newCall(request).execute();
+    } catch (IOException e) {
+      response = null;
+      System.err.println(e);
+    }
+
+    return response;
+  }
+
+  // checks if a value is an integer
+  private static boolean isInteger(String value) {
+    try {
+      Integer.parseInt(value);
+      return true; // value is int
+    } catch (Exception e) {
+      return false; // value isn't int
+    }
+  }
```

Comment:
My intelliJ tells me this method is unused, if that is the case we should remove it.

Reply:
done.